CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ Compatible with the v2.5.0 version of nebula-graph
 	- add TimeWrapper/DateTimeWrapper type to get timezone obtained from the server to calculate the local time
 	- Session supports reconnecting to different graph services
 - bugfix
-	- fix the interface valuesof Relationship and modify the interface propertysto propertieshttps://github.com/vesoft-inc/nebula-python/pull/113
+	- fix the interface values of Relationship and modify the interface properties to properties https://github.com/vesoft-inc/nebula-python/pull/113
```

Comment:
@greyli explained in https://github.com/vesoft-inc/nebula-python/pull/173#issuecomment-966016705 that this changelog entry was trying to say it was changing `propertys` to `properties`.

Would this be acceptable? ([[sic]](https://www.merriam-webster.com/dictionary/sic) is used to indicate it reproduces something as originally written (as opposed to how it should be written))
```suggestion
	- fix the interface values of Relationship and modify the interface `propertys` [sic] to `properties` https://github.com/vesoft-inc/nebula-python/pull/113
```

Reply:
Totally!